Assets/MRTK/Extensions/SceneTransitionService/AssemblyInfo.cs

```diff
@@ -8,3 +8,7 @@
 
 [assembly: AssemblyProduct("Microsoft® Mixed Reality Toolkit Extensions")]
 [assembly: AssemblyCopyright("Copyright © Microsoft Corporation")]
+
+[assembly: UnityEngine.Scripting.Preserve]
```

Comment:
This still has a preserve marked on it

Reply:
It might be worth doing another pass (i.e. grep -ri preserve *) to make sure we fully clean this.

Reply:
i missed one file 😄

Reply:
fixed